distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -223,7 +214,7 @@ type Vector a
     reduce self function =
         case self.not_empty of
             True -> if self.length == 1 then self.unsafe_at 0 else
-                f = acc -> ix -> function acc (self.storage.at ix)
+                f = acc -> ix -> function acc (self.at ix)
```

Comment:
Changed to `unsafe_at` in 046ea846614.

However I'd like to get rid of `unsafe_at`, I doubt it has any benefits (that wouldn't be approachable otherwise) and only clutters the `Vector` API. Reported as [#183000876](https://www.pivotaltracker.com/n/projects/2539304/stories/183000876).